src/languageserverinstance.jl

```diff
@@ -114,9 +116,31 @@ end
 
 function setdocument!(server::LanguageServerInstance, uri::URI2, doc::Document)
     server._documents[uri] = doc
+    # Add possible workspace packages
+    path = uri2filepath(uri._uri)
+    for wk_folder in server.workspaceFolders
+        if startswith(path, wk_folder) && normpath(wk_folder) == normpath(server.env_path)
```

Comment:
I don't fully understand the logic here. Is the idea that only if one has activated the package as the env, will any of this work?